File: 1. Model/LocalDeEvento.py


class LocalDeEvento:

    def __init__(self, cidade, bairro, rua, pontoEncontro):
        self.cidade        = cidade
        self.bairro        = bairro
        self.rua           = rua
        self.pontoEncontro = pontoEncontro





    #  - - - - - - - - - - GETS & SETS - - - - - - - - - -
    ##GETS
    def getCidade(self):
        return self.cidade

    def getBairro(self):
        return self.bairro

    def getRua(self):
        return self.rua

    def getPontoEncontro(self):
        return self.pontoEncontro


    ##SETS
    def setCidade(self, c):
        self.cidade = c

    def setBairro(self, b):
        self.bairro = b

    def setRua(self, r):
        self.rua = r

    def setPontoEncontro(self, pe):
        self.pontoEncontro = pe
        
    # - - - - - - - - - - //// /  //// - - - - - - - - - -

File: model/LocalDeEvento.java
package model;



public class LocalDeEvento {

	private String cidade;
	private String bairro;
	private String rua;
	private String pontoEncontro;



	public LocalDeEvento(String cidade, String bairro, String rua, String pontoEncontro) {
		this.cidade 	   = cidade;
		this.bairro 	   = bairro;
		this.rua 		   = rua;
		this.pontoEncontro = pontoEncontro;
	}



	// Gets
	public String getCidade() {
		return this.cidade;
	}

	public String getBairro() {
		return this.bairro;
	}

	public String getRua() {
		return this.rua;
	}

	public String getPontoEncontro() {
		return this.pontoEncontro;
	}



	// Sets
	public void setCidade(String c) {
		this.cidade = c;
	}

	public void setBairro(String b) {
		this.bairro = b;
	}

	public void setRua(String r) {
		this.rua = r;
	}

	public void setPontoEncontro(String pe) {
		this.pontoEncontro = pe;
	}

}
